packages/jest-emotion/package.json

```diff
@@ -20,6 +22,9 @@
     "css": "^2.2.1",
     "specificity": "^0.4.1"
   },
+  "peerDependencies": {
```

Comment:
This should be an optional peer dependency, to mark is as such you need to add `peerDependenciesMeta` similar to this one - https://github.com/emotion-js/emotion/blob/456be9a602d7d0bac291617f69f59f5ed30d1b84/packages/core/package.json#L38-L42